test/lib/time.rb

```diff
@@ -71,6 +85,21 @@ def minutes_conversion(minutes)
     "#{hours}:#{rest}"
   end
 
+  def inner_sections?(minutes_arr, arr, time)
+    minutes_arr[arr][time + 1] >= minutes_arr[arr + 1][time + 1] && minutes_arr[arr][time] <= minutes_arr[arr + 1][time]
```

Comment:
What is mean 1?

Reply:
I am comparing the second variable of the first array with the second variable of the second array and the first variable of the first array with the first variable of the second array. 1 increases the values of time and arr.